fern/pages/api-definition/openapi/extensions.mdx

```diff
@@ -558,6 +558,31 @@ shown in the third tab.
     ```
 </CodeBlocks>
 
+## Embedding extensions
+
+If instead of overlaying your extensions within an overrides file, as mentioned above. Certain frameworks that generate OpenAPI specs make it easy to embed extensions directly from code.
+
+### FastAPI
+
+FastAPI allows you to add in extra OpenAPI configuration directly within your route, through the use of the `openapi_extra` parameter.
+Below, we've annotated a "good" route within FastAPI that has it's typings as well as Fern extensions to assist in naming.
+
+```python
+@router.post(
+    "/your/post/endpoint",
+    response_model=YourResponseModel,            #  <-- with FastAPI, it is important to specify your response model so that it comes through to the OpenAPI spec
+    summary="Get some response for your req",    #  <-- if you'd like to add a description to your endpoint, you can do so here
+    openapi_extra={                              #  <-- finally, you can add in your Fern extensions here, these extensions will produce SDK code that looks something like: `client.endpoints.create(...)` in python
+        "x-fern-sdk-method-name": "create",
+        "x-fern-sdk-group-name": "endpoints",
+        "x-fern-availability": "beta",
+    },
+)
+async def your_post_endpoint(
+    payload: YourRequestModel,
+) -> YourResponseModel:
```

Comment:
i think itll be easier to put all fastapi related stuff in one place

Reply:
I thought it made more sense to live here since this is the section on extensions and link out to here from the fastapi page (which is what it is now)

Reply:
In my opinion it's just easier if there's already a dedicated page on the topic to include the content there, then link to it from the framework page

that way regardless of entry point you can find what you're looking for

Reply:
I was just on a call and had to hunt for `fastapi` in the preview link -- Definitely think a page centered around FastAPI even if it is repetitive is easier for users + demoers